docs/EN_US/ContainerizedHPCC/ContainerizedMods/ConfigureValues.xml

```diff
@@ -0,0 +1,1255 @@
+        enabled, the component resources must be defined such that the pod has
+        enough resources to handle the resource demands of all the those jobs
+        to be able to run in parallel.</para>
+
+        <para>With useChildProcess enabled it could be rather expensive in
+        most cloud pricing models, and rather wasteful if there aren't any
+        jobs running. Instead you can set this
+        <emphasis>useChildprocess</emphasis> to false (the default) to start
+        your pod with only the required memory for the job which will be
+        disposed of when done. Now this model also has overheard, perhaps 20
+        seconds to a minute to spawn the Kubernetes cluster to process the
+        job. Which may not be ideal for an environment which is sending
+        several small jobs, but rather larger jobs which would minimize the
+        effect of the overhead in starting the Kubernetes cluster. Setting
+        <emphasis>useChildProcess</emphasis> to false better allows for the
+        possibility of dynamic scaling. For jobs which would take a long while
+        to compile, the extra (start up) overhead is minimal, and that would
+        be the ideal case to have the <emphasis>useChildProcess</emphasis> as
+        false. Setting <emphasis>useChildProcess</emphasis> to false only
+        allows 1 pod per compile.</para>
+
+        <para><emphasis role="bold">maxActive</emphasis>: The maximum number
+        of jobs that can be run in parallel. Again use caution because each
+        job will need enough memory to run. For instance, if
+        <emphasis>maxActive</emphasis> is set to 2000, you could submit a very
+        big job and in that case spawn some 2000 jobs using a considerable
+        amount of resources, which could potentially run up a rather expensive
+        compilation bill, again depending on your cloud provider and your
+        billing plan.</para>
+      </sect3>
+
+      <sect3 id="ValYML_Sasha">
+        <title>Sasha</title>
+
+        <para>Sasha is an outlier as it is a dictionary type structure and not
+        a list. You can't have more than one archiver or dfuwu-archiver as
+        that is a value limitation, you can choose to either have the service
+        or not (set the 'disabled' value to true). The Sasha component is an
+        exception to the norm for HPCC Systems components.</para>
+      </sect3>
+
+      <sect3 id="ValYML_Thor">
+        <title>Thor</title>
+
+        <para>Thor instances run dynamically. As do the other components in
+        containers. In Thor's case, a collection of pods is spawned (manager +
+        N workers) when jobs are queued to it. When idle there are no worker
+        (or manager) pods running.</para>
+
+        <para>In the case of Thor (components) there are similarities with the
+        ECLCC Server. Thor is also a list, and if that list has a value
+        of</para>
+
+        <programlisting>-name: thor </programlisting>
```

Comment:
not sure how these examples relate to the surrounding text.

Reply:
It does seem a bit awkward. 
Would more of the Thor component list in the values.yaml make more sense?    

thor:
- name: thor
  prefix: thor
  numWorkers: 2
  maxJobs: 4
  maxGraphs: 2

Reply:
I think it is more that the sentence doesn't make sense.

"and if that list has a value of

        -name: thor

or

        -name: thor_50

If you have..."

That isn't really a sentence. and I don't understand what it means..